README.md

```diff
@@ -23,21 +23,22 @@
 
 ## Background
 [AWS SES](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/Welcome.html)
-provides both an
+and [Pinpoint](https://docs.aws.amazon.com/sdk-for-go/api/service/pinpointemail/) provide an
 [API](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/send-email-api.html)
 and an [SMTP interface](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/send-email-smtp.html)
 to send emails.
 
 The SMTP interface is useful for applications that must use SMTP to send emails,
 but it requires providing a set of
-[SES SMTP Credentials](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/smtp-credentials.html).
+[SES SMTP Credentials](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/smtp-credentials.html), 
+and these. credentials are shared to the Pinpoint API.
```

Comment:
I think it's better to provide links to each specific documentation:

```
The SMTP interface (SES/Pinpoint) is useful for applications that must use SMTP to send emails,
but it requires providing a set of SMTP credentials:
* [SES SMTP Credentials](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/smtp-credentials.html)
* [Pinpoint SMTP Credentials](https://docs.aws.amazon.com/pinpoint/latest/userguide/channels-email-send-smtp.html#channels-email-send-smtp-credentials)
```

Reply:
Done.